src/ContainerRegistry/ContainerRegistry/Commands/UpdateAzureContainerRegistry.cs

```diff
@@ -104,7 +109,7 @@ public override void ExecuteCmdlet()
 
             if (ShouldProcess(Name, "Update Container Registry"))
             {
-                var registry = RegistryClient.UpdateRegistry(ResourceGroupName, Name, adminUserEnabled, Sku, storageAccountId, tags);
+                var registry = RegistryClient.UpdateRegistry(ResourceGroupName, Name, adminUserEnabled, Sku, storageAccountId, NetworkRuleSet, tags);
                 WriteObject(new PSContainerRegistry(registry));
             }
         }
```

Comment:
For phase two, I suggest expose:

- AllowPublicNetworkAccess
-  Enable/Disable data endpoint